recipes/_compression_libs.rb

```diff
@@ -24,7 +24,11 @@
 # Everybody gets snappy
 case node['platform_family']
 when 'debian'
-  pkgs += ['libsnappy1', 'libsnappy-dev']
+  pkgs += if node['platform_version'] >= '16'
```

Comment:
it's better to compare as integers, ie ``node['platform_version'].to_i >= 16``.  Otherwise its just comparing lexicographically where for example '9.0' > '16.0'